scripts/build-shotcut.sh

```diff
@@ -410,6 +417,9 @@ function set_globals {
     if test "$ENABLE_BIGSH0T" = 1 ; then
         SUBDIRS="$SUBDIRS bigsh0t"
     fi
+    if test "$ENABLE_ZIMG" = 1 ; then
+        SUBDIRS="$SUBDIRS zimg"
```

Comment:
This needs to be `SUBDIRS="zimg $SUBDIRS"` so that FFmpeg build can see it.

Reply:
I can fix this order un SUBDIRS in merge.  I made changes for the windows build and testing it now.